.asf.yaml

```diff
@@ -18,6 +18,7 @@ github:
     v2: { }
     v3: { }
     v4: { }
+    release-drafter: { }
```

Comment:
add a comment: "waiting for PR merge https://github.com/release-drafter/release-drafter/pull/1451"
would clarify the unusual intent of this branch

Reply:
done